planets.rb

```diff
@@ -0,0 +1,152 @@
+
+class SolarSystem
+  attr_accessor :planets, :ranked_planets
+
+  # Stores an array of hashes of info on planets
+  def initialize (planets)
+    @planets = planets
+    @ranked_planets = []
+  end
+
+  # Cycles through list of planets
+  # Returns numbered list of planets, w/ respect to sun proximity
+  def cycle_through_system
```

Comment:
This method should just return a list of planet names, not add elements to another instance variable.  If you hadn't recreated the `SolarSystem` each iteration of the main loop you would end up with double and triple entries for each planet in `@ ranked_planets`.